### PR TITLE
Sequelize

### PR DESCRIPTION
**Pull request by eduardgasparyan**

### CODE REVIEW

File: routers/userRouting.js [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+  const email = data.email;
+  return db.createOne(username, password, email);
+});
+
+router.post('/login', async ctx => {
+  const bodyParsed = ctx.request.body;
+  const data = bodyParsed;
+  const username = data.username;
+  const password = data.password;
+  return db.login(username, password);
```
Comment (Karlen92) [inline routers/userRouting.js]: use `{username, password}` syntax for you functions

File: services/userData.js [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+exports.login = async (username, password) => {
+  const user = await sequelize.User.findOne({
+    where: {
+      username: username,
+    },
+  });
+  const hashedPassword = user.password;
+  const newHashedPassword = await bcrypt.hash(password, user.salt);
+  if (hashedPassword === newHashedPassword) {
+    return console.log('logged');
```
Comment (Karlen92) [inline services/userData.js]: you cannot return console.log, console.log is just logging, just for information, it does not have any sense, please revise.

File: services/userData.js [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+
+exports.deleteOne = async param => {
+  await sequelize.User.destroy({
+    where: { id: param },
+    force: true,
+  });
+  return sequelize.User.findAll();
+};
+
+exports.updateAge = async (username, age) => {
```
Comment (Karlen92) [inline services/userData.js]: You don't need to write an update function for every user attribute, make 1 update function, which will be able to update any property, besides come fields, for example, password and email.

File: services/userData.js [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+  });
+  await sequelize.User.create(user);
+  return console.log(
+    await sequelize.User.findOne({
+      where: { id: user.id },
+    }),
+  );
+};
+
+exports.addAgeColumn = async () => {
```
Comment (Karlen92) [inline services/userData.js]: Remove this migration cannot be called from the code

**Review by Karlen92 — CHANGES_REQUESTED** [review]

File: routers/userRouting.js (hunk 1 above)
Comment (eduardgasparyan) [reply routers/userRouting.js]: `router.post('/register', async ctx => {
  const bodyParsed = ctx.request.body;
  const data = bodyParsed;
  return db.createOne(data);
});`  I'm transferring data object, where are all we got from JSON and in our createOne function we use information from our transferred object.

**Pull request merged**